CHANGELOG.rst

```diff
@@ -17,6 +17,7 @@ Changes are ordered reverse-chronologically.
  - Fixed logging on Windows
  - Added ability for coaches to make copies of exams
  - Added icon next to language-switching functionality
+ - Languages: English, Arabic, Farsi, French, Hindi, Spanish, Swahili, and Urdu
```

Comment:
Awesome, will make sure to check in the Perseus translations that they're included.